html-templates/simple.mustache

```diff
@@ -0,0 +1,332 @@
+		}
+
+ 		h1, h4 {
+			text-align: center;
+		}
+
+		#footer {
+		    padding-top: 10px;
+		    padding-bottom: 10px;
+			background-color: #e0e0e0;
+			font-size: 1.5em;
+			text-align: center;
+		}
+
+		#reportTitle {
+			margin: 25px;
+		}
+
+		#summaryFiles {
+			margin-top: 25px;
+		}
+	</style>
+	<script type="text/javascript" language="javascript" src="https://code.jquery.com/jquery-3.5.1.js"></script>
+	<script type="text/javascript" language="javascript"
+		src="https://cdn.datatables.net/1.10.21/js/jquery.dataTables.min.js"></script>
+	<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/rowgroup/1.1.2/js/dataTables.rowGroup.min.js"></script>
+    <script type="text/javascript" language="javascript" src="https://cdn.jsdelivr.net/npm/chart.js@2.9.3/dist/Chart.min.js"></script>
+	<script type="text/javascript" class="init">
+
+	// BEGIN - Placeholders filled in by the scanner
+	const violations = {{{violations}}};
+	// END - Placeholders filled in by the scanner
+
+	// Global DataTable displayed in the report
+	var dataTable = null;
+
+	// Color if the severity is greater than 5
+	const unknownSeverityColor = '#cccccc';
+
+	// Background colors for severity
+    // Palette from https://www.w3schools.com/colors/colors_palettes.asp
+    // TODO: Accessibility
+	function getSeverityColor(severity) {
+		switch (parseInt(severity)) {
+			case 1:
+				return '#d96459';
+			case 2:
+				return '#f2ae72';
+			case 3:
+				return '#f2e394';
+			case 4:
+				return '#588c7e';
+			case 5:
+				return '#96ceb4';
+			default:
+				return '#cccccc';
+		}
+	}
+
+	// Convert a map of severity->count to a structure consumable by the chart library
+	// options.labelFormatter: function(severity, count) that returns a label
+	function sevMapToChartData(sevMap, options) {
+		var dataset = {
+			data: [],
+			backgroundColor: []
+		};
+
+		var dataLabels = [];
+		// Sort the entries by key order
+		sevMap = new Map([...sevMap.entries()].sort());
+		for (var severity of sevMap.keys()) {
+			var count = sevMap.get(severity);
+			dataset.data.push(count);
+			dataset.backgroundColor.push(getSeverityColor(severity));
+			if (options && options.labelFormatter) {
+				dataLabels.push(options.labelFormatter(severity, count));
+			} else {
+				dataLabels.push(`Sev${severity}`);
+			}
+		}
+
+		return {
+			datasets: [dataset],
+			labels: dataLabels
+		};
+	}
+
+	// Creates a horizontal bar chart below a grouping of rows. Called
+	// asynchronously as the rows are added/removed by the search feature.
+	function populateRowGroupChart(id, summary) {
+		const ctx = $(`#${id}`);
+		if (!ctx.length) {
+			// This can happen when the user quickly types in the search field
+			return;
+		}
+
+		const data = sevMapToChartData(summary);
+
+		new Chart(ctx, {
+			type: 'horizontalBar',
+			data: data,
+			options: {
+				aspectRatio: 20,
+				animation: {
+					duration: 0
+				},
+				legend: {
+					display: false
+				},
+				scales: {
```

Comment:
This is the change to the axes, attaching two screenshots to show the diff. The one with 3 visible bars is the new change. 

![Selection_003](https://user-images.githubusercontent.com/18196574/92153722-944d4700-edd9-11ea-9496-557cf93305d4.png)
![Selection_002](https://user-images.githubusercontent.com/18196574/92153742-9911fb00-edd9-11ea-9a7b-22d64252b94e.png)